packages/components/collapsible-panel/src/collapsible-panel.spec.js

```diff
@@ -159,94 +159,72 @@ describe('getPanelContentId', () => {
 });
 
 describe('aria attributes', () => {
-  it('should have a valid aria-controls correspondence', () => {
+  const renderPanel = props => {
     const rendered = render(
       <CollapsiblePanel
         id="example"
         header="Header"
         onToggle={jest.fn()}
-        isClosed={false}
+        {...props}
       >
         Children
       </CollapsiblePanel>
     );
+    const getPanelHeader = () => rendered.container.querySelector('button');
 
-    const panelContentId = CollapsiblePanel.getPanelContentId('example');
+    const panelContentId = CollapsiblePanel.getPanelContentId(
+      props.id || 'example'
+    );
+    const getPanelContent = () =>
```

Comment:
Nit: get* throws in rtl. That's why I felt `query` (which doesn't) wording wise was more apt.

Reply:
I only didn't call these `query` because they don't accept any args to _query by_, they "automatically" **get** the element we want without exposing the query 😄 Hope that's okay

Reply:
Totally. Thanks. You're a beach bum!